neco-server-exporter/e2e/bpf_test.go

```diff
@@ -0,0 +1,45 @@
+package e2e
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func testBPFCollector() {
```

Comment:
This function has not only tests for bpf-collector but also tests for the exporter.
How about spliting the context into exporter and bpf-collector?